VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
```

Comment:
How about just removing the `try` `catch` block from here, and handling it in `onException()`?